### PR TITLE
Issue #590: Fretboard: Blog post

### LINKED ISSUE #590: Fretboard: Blog post

### PR DESCRIPTION
**Pull request by fercarcedo**
Closes #590

### CODE REVIEW

File: docs/_posts/2018-08-10-fretboard.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Fretboard: A/B Testing Framework for Android"
```
Comment (pocmo) [inline docs/_posts/2018-08-10-fretboard.md]: Let's add GSoC to the title:

"☀️ Google Summer of Code: Fretboard - A/B Testing Framework for Android"

Comment (fercarcedo) [reply docs/_posts/2018-08-10-fretboard.md]: Yes, I forgot about that. Now it's added :)

**Comment by pocmo** [comment]
I'll try to quickly add an author configuration to jekyll so that we can add you as author of this posting. :)

File: docs/_posts/2018-08-10-fretboard.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: ":sunny: Google Summer of Code: Fretboard - A/B Testing Framework for Android"
+date: 2018-08-10 14:48:00 +0200
+---
```
Comment (pocmo) [inline docs/_posts/2018-08-10-fretboard.md]: Alright, now we can add the author. :)

Comment (fercarcedo) [reply docs/_posts/2018-08-10-fretboard.md]: Done :)

**Pull request merged**